data/lakshmishreea122003.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Lakshmi Shree A",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "ML and DevOps enthusiast| Java| Python| Web development| Open Source 🚀",
+  "avatar":"https://github.com/lakshmishreea122003.png",
```

Comment:
In our recent update, we no longer need the `avatar` key. So please remove this from your json file. 
```suggestion
```